src/main/java/net/sf/jabref/logic/undo/UndoRedoEvent.java

```diff
@@ -0,0 +1,10 @@
+package net.sf.jabref.logic.undo;
+
+
+public class UndoRedoEvent extends AddUndoEvent {
```

Comment:
I'll add comments. Quickly: AddUndoEvent when something is added to the
undo manager (update actions), UndoRedoEvent when something is
undone/redone (update actions and possibly entry editor).